Unexpected close loading wallet

**Describe the bug**
Having a list of deterministic wallets, taking the seed of one of them to load a bip44 wallet with this seed make the application to close unexpectedly.

**Environment information:**

- OS: GNU/Linux
- Platform: Linux 5.2.0-3-amd64 x86_64

**Steps to Reproduce**
Steps to reproduce the behavior:

1. Go to the application UI
2. Create a new deterministic wallet
3. Load a new bip44 wallet with the same seed as the previous.
4. See the stack trace...

**Actual behavior**
```
[2019-12-09T18:15:55-05:00] INFO [Skycoin Wallet]: Getting wallet set from wallet directory
[2019-12-09T18:15:55-05:00] INFO [Skycoin Wallet]: Getting Skycoin local wallet
[2019-12-09T18:15:55-05:00] ERROR [wallet]: Load: file.LoadJSON failed error="read /home/adacosta/.skycoin/wallets: is a directory" filename="/home/adacosta/.skycoin/wallets"
[2019-12-09T18:15:55-05:00] ERROR [Skycoin Wallet]: Call to wallet.Load(filename) inside GetWallet failed. error="read /home/adacosta/.skycoin/wallets: is a directory" filename="/home/adacosta/.skycoin/wallets"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0xc3b624]

goroutine 1 [running, locked to thread]:
github.com/fibercrypto/fibercryptowallet/src/models.(*WalletManager).updateAddresses(0xc00022b900, 0x0, 0x0)
        github.com/fibercrypto/fibercryptowallet/src/models/walletsManager.go:137 +0x114
github.com/fibercrypto/fibercryptowallet/src/models.(*WalletManager).getAddresses(0xc00022b900, 0x0, 0x0, 0x5d0ef60, 0x1200000, 0xc000103c70)
        github.com/fibercrypto/fibercryptowallet/src/models/walletsManager.go:672 +0xa5
github.com/fibercrypto/fibercryptowallet/src/models.(*AddressesModel).updateModel(0xc00030c150, 0x0, 0x0)
        github.com/fibercrypto/fibercryptowallet/src/models/addressesModel.go:196 +0x41
github.com/fibercrypto/fibercryptowallet/src/models.callbackAddressesModel72d3a2_UpdateModel(0x5a566a0, 0x51bbb02, 0x0)
        github.com/fibercrypto/fibercryptowallet/src/models/moc.go:22987 +0xa1
github.com/fibercrypto/fibercryptowallet/src/models._cgoexpwrap_b6d9ad687100_callbackAddressesModel72d3a2_UpdateModel(0x5a566a0, 0x51bbb02, 0x0)
        _cgo_gotypes.go:35389 +0x3f
github.com/therecipe/qt/widgets._Cfunc_QApplication_QApplication_Exec(0xc000000000)
        _cgo_gotypes.go:1033 +0x49
github.com/therecipe/qt/widgets.(*QApplication).Exec(...)
        github.com/therecipe/qt/widgets/widgets-minimal.go:1009
main.main()
        github.com/fibercrypto/fibercryptowallet/main.go:60 +0x2ba
make: *** [Makefile:100: run] Error 2
```

**Expected behavior**
A new wallet should be loaded

--- a/src/coin/skycoin/models/wallet.go
+++ b/src/coin/skycoin/models/wallet.go
@@ -31,7 +31,6 @@ const (
 	CoinHour       = params.CoinHoursTicker
 	CalculatedHour = params.CalculatedHoursTicker
 
-
 	walletExt             = ".wlt"
 	WalletTimestampFormat = "2006_01_02"
 
@@ -884,7 +883,6 @@ func (wltSrv *SkycoinLocalWallet) CreateWallet(label string, seed string, wltTyp
 	pwdCtx.SetValue(core.StrMethodName, "CreateWallet")
 	pwdCtx.SetValue(core.StrWalletLabel, label)
 	password, err := pwd("Insert Password", pwdCtx)
-
 	if err != nil {
 		logWallet.WithError(err).Fatal("Something was wrong entering the password")
 		return nil, err

--- a/src/models/walletsManager.go
+++ b/src/models/walletsManager.go
@@ -680,7 +680,6 @@ func (walletM *WalletManager) createEncryptedWallet(seed, label, wltType, passwo
 	pwd := util.ConstantPassword(password)
 	// NOTE: No easy way to get plain passwords in memory
 	password = ""
-	fmt.Println("WALLET TYPE ", wltType)
 	wlt, err := walletM.WalletEnv.GetWalletSet().CreateWallet(label, seed, wltType, true, pwd, scanN)
 	if err != nil {
 		logWalletManager.WithError(err).Error("Couldn't create encrypted wallet")
@@ -695,13 +694,13 @@ func (walletM *WalletManager) createEncryptedWallet(seed, label, wltType, passwo
 func (walletM *WalletManager) createUnencryptedWallet(seed, label, wltType string, scanN int) *QWallet {
 	logWalletManager.Info("Creating encrypted wallet")
 	pwd := util.EmptyPassword
-
 	wlt, err := walletM.WalletEnv.GetWalletSet().CreateWallet(label, seed, wltType, false, pwd, scanN)
 	if err != nil {
 		logWalletManager.WithError(err).Error("Couldn't create unencrypted wallet")
 		return nil
 	}
-	logWalletManager.Info("Created encrypted wallet")
+	logWalletManager.Info("Created unencrypted wallet")
+
 	return fromWalletToQWallet(wlt, false)
 
 }

--- a/src/models/walletsModel.go
+++ b/src/models/walletsModel.go
@@ -25,22 +25,22 @@ type WalletModel struct {
 	_ map[int]*core.QByteArray `property:"roles"`
 	_ []*QWallet               `property:"wallets"`
 
-	_ func(*QWallet)                                                                    `slot:"addWallet"`
-	_ func(row int, name string, encryptionEnabled bool, sky string, coinHours string)  `slot:"editWallet"`
-	_ func(row int)                                                                     `slot:"removeWallet"`
-	_ func([]*QWallet)                                                                  `slot:"loadModel"`
-	_ func([]*QWallet)                                                                  `slot:"updateModel"`
-	_ int                                                                               `property:"count"`
+	_ func(*QWallet)                                                                   `slot:"addWallet"`
+	_ func(row int, name string, encryptionEnabled bool, sky string, coinHours string) `slot:"editWallet"`
+	_ func(row int)                                                                    `slot:"removeWallet"`
+	_ func([]*QWallet)                                                                 `slot:"loadModel"`
+	_ func([]*QWallet)                                                                 `slot:"updateModel"`
+	_ int                                                                              `property:"count"`
 }
 
 type QWallet struct {
 	core.QObject
-	_ string  `property:"name"`
-	_ int     `property:"encryptionEnabled"`
-	_ string  `property:"sky"`
-	_ string  `property:"coinHours"`
-	_ string  `property:"fileName"`
-	_ bool    `property:"expand"`
+	_ string `property:"name"`
+	_ int    `property:"encryptionEnabled"`
+	_ string `property:"sky"`
+	_ string `property:"coinHours"`
+	_ string `property:"fileName"`
+	_ bool   `property:"expand"`
 }
 
 func (walletModel *WalletModel) init() {

--- a/src/models/walletsModel.go
+++ b/src/models/walletsModel.go
@@ -176,6 +176,9 @@ func (walletModel *WalletModel) roleNames() map[int]*core.QByteArray {
 
 func (walletModel *WalletModel) addWallet(w *QWallet) {
 	logWalletsModel.Info("Add Wallet")
+	if w.Pointer() == nil {
+		return
+	}
 	walletModel.BeginInsertRows(core.NewQModelIndex(), len(walletModel.Wallets()), len(walletModel.Wallets()))
 	qml.QQmlEngine_SetObjectOwnership(w, qml.QQmlEngine__CppOwnership)
 	walletModel.SetWallets(append(walletModel.Wallets(), w))

--- a/src/ui/Dialogs/DialogAddLoadWallet.qml
+++ b/src/ui/Dialogs/DialogAddLoadWallet.qml
@@ -31,15 +31,15 @@ Dialog {
             if (mode === CreateLoadWallet.Load){
                 scanA = 10
             }
-            walletModel.addWallet(walletManager.createEncryptedWallet(seed, name,textFieldPassword.text, comboBoxWalletType.model[comboBoxWalletType.currentIndex].value, scanA))
+            
+            walletModel.addWallet(walletManager.createEncryptedWallet(seed, name,textFieldPassword.text, comboBoxWalletType.model[comboBoxWalletType.currentIndex].name, scanA))
             
         } else{
             
             if (mode === CreateLoadWallet.Load){
                 scanA = 10
             }
-            
-            walletModel.addWallet(walletManager.createUnencryptedWallet(seed, name, comboBoxWalletType.model[comboBoxWalletType.currentIndex].value, scanA))
+            walletModel.addWallet(walletManager.createUnencryptedWallet(seed, name, comboBoxWalletType.model[comboBoxWalletType.currentIndex].name, scanA))
         }
     }
 

